### DISCUSSION TITLE
Получение координат при клике на карте

### DISCUSSION THREAD

**Discussion by Dior1726** [yandex-maps-unofficial/vue-yandex-maps#165] [Q&A (2.x)] [answered]
![image](https://github.com/yandex-maps-unofficial/vue-yandex-map/assets/48237769/31760874-5875-4ba2-a8e8-6da36e75d155)
![image](https://github.com/yandex-maps-unofficial/vue-yandex-map/assets/48237769/ae750b77-6d78-4692-a3fc-1fa7d6f8b917)
![image](https://github.com/yandex-maps-unofficial/vue-yandex-map/assets/48237769/ba648f04-53eb-4768-a1d8-5d6723db2e0d)
![image](https://github.com/yandex-maps-unofficial/vue-yandex-map/assets/48237769/2df0c64e-450a-4f85-891e-d7acce4b9e4d)

### Не получается взять координаты при клике на карте.

**Comment by daniluk4000** [comment, marked as answer]
Добрый день! Вы ошиблись репозиторием, перенес в актуальный и в дискуссии.

Данный `@click` работает не так, как вы ожидаете - он просто повесился на компонент Карты силами Vue. Чтобы отловить клик по карте от Яндекса, вам необходимо использовать [YandexMapListener](https://yandex-maps-unofficial.github.io/vue-yandex-maps/components/listener.html).

**Reply by Dior1726** [reply]
Спасибо!

**Reply by MaGaKZ** [reply]
Мучился с этим пару дней спасибо!
